server/tests/client/decentralized.spec.ts

```diff
@@ -32,7 +33,7 @@ function test (
         getClient(Client, server, TASK),
         getClient(Client, server, TASK)
       ]))
-
+      const preparators = users.map(u => new ClearPreparator(u, TASK))
```

Comment:
add some comments / docstring what preparators are and why needed

Reply:
If I recall correctly, after a discussion with Hugo and Lucas, we figured that splitting aggregation strategies into a preparator/aggregator maked more sense than having a unique aggregator in order to reuse code between decentralized and federated. Preparator prepares the extra processing that has to be done on the client so that Aggregator only handles the aggregation (or norm checks in case of clipping).

If we take secure aggregation in federated for example, the preparator would handle all client-side pre-computation (shares of 0 generation and adding them to the weights), while the aggregator would handle the server-side computation, so in this case, just averaging.

In decentralized, preparator/aggregator are both part of the client, while in federated the preparator/aggregator are split between client/server. That enables code reuse of preparators between decentralized and federated.  (correct me if I'm wrong on the idea here @LucasTrg )